dm/tests/all_mode/run.sh

```diff
@@ -377,6 +377,9 @@ function run() {
 	# use sync_diff_inspector to check full dump loader
 	check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
 
+	# check create view(should be skipped by func `skipSQLByPattern`) will not stop sync task
+	run_sql_source1 "create view all_mode.t1_v as select * from all_mode.t1 where id=0;"
```

Comment:
please add a check about task is not interrupted. Currently the test will fail at line 449, that's too far